TLM/TLM/Patch/_TransportLineAI/StartPathFindPatch.cs

```diff
@@ -75,6 +76,7 @@ public static class StartPathFindPatch {
                 new CodeInstruction(OpCodes.Ldloc_S, 10),
                 new CodeInstruction(OpCodes.Ldarg_S, 4),
                 new CodeInstruction(OpCodes.Ldarg_S, 5),
+                new CodeInstruction(OpCodes.Ldarg_S, 6),
```

Comment:
At some point in future we should get arguments by name rather than position

Reply:
I have code in my mod to do this.

Reply:
yeah, might be improved, but you can't change the order or add new params to the method definition (ok.. you could if you try hard enough - don't 😂 ) :)